pom.xml

```diff
@@ -9,9 +9,7 @@
     <packaging>pom</packaging>
 
     <name>ecoCode Sonar Plugins Project</name>
-    <description>Les règles s'appuient sur l'édition 3 du livre "Ecoconception Web / Les 115 bonnes pratiques" =&gt;
-        https://collectif.greenit.fr/ecoconception-web/115-bonnes-pratiques-eco-conception_web.html
-    </description>
+    <description>Provides rules to reduce the environmental footprint of your programs</description>
```

Comment:
I think we can do this changes for all SonarQube plugins?

Reply:
I agree with you, but it seems that this description is used for all plugins excepted javascript plugin that already define a good description : 

ecoCode JavaScript Sonar PluginProvides rules to reduce the environmental footprint of your JavaScript programs